Client/src/components/Shared/HeroBillboard/HeroBillboard.jsx

```
@@ -0,0 +1,102 @@
+            <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-blue-500 to-blue-700 rounded-full shadow-md"></span>
+            <span className="absolute inset-0 w-full h-full bg-white/15 rounded-full blur-[2px]"></span>
+            <span className="absolute inset-0 w-full h-full bg-blue-600 rounded-full transform transition-transform group-hover:scale-105"></span>
+            <span className="relative flex items-center">
+              <Play size={20} className="mr-2" /> Demo
+            </span>
+          </button>
+          <button 
+            className="relative group flex items-center bg-gray-800 px-6 py-3 rounded-full font-semibold text-white hover:bg-gray-700 transition-colors border border-gray-700"
+            type="button"
+          >
+            <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-gray-800 to-gray-700 rounded-full shadow-md"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)